packages/trader/src/Modules/Trading/Components/Elements/PayoutPerPoint/payout-per-point.scss

```
@@ -0,0 +1,82 @@
+.payout-per-point-input {
+    padding-bottom: 0rem;
+    .popover_wrapper_class {
+        margin-top: 0.3rem;
+    }
+    .actions-wrapper {
+        margin-top: 0.6rem;
+        display: flex;
+        justify-content: space-between;
+        .distance-to-current-spot {
+            display: flex;
+            .barrier-value {
+                margin-bottom: 0.8rem;
+                background-color: var(--general-section-1);
+                border: none;
+                border-bottom: 1px dotted var(--text-general);
+                padding: 0;
+                display: flex;
+                cursor: pointer;
+                .indicator-icon {
+                    margin-left: 0.2rem;
+                    margin-top: 0.3rem;
+                }
+            }
+        }
+    }
+}
+
+.payout-per-point-mobile {
+    background: var(--general-section-1);
+    width: 100%;
+    border-radius: 0.4rem;
+    display: flex;
+    height: 250px;
+    padding: 0.6rem 0.8rem;
+    flex-direction: column;
+    &__header {
+        padding: 0.8rem;
+        display: flex;
+        &__title {
+            display: flex;
+            gap: 0;
+        }
+        .popover-icon {
+            margin-left: 0.8rem;
```

```suggestion
            margin-inline-start: var(--core-spacing-400);
```
it's better to use css logical properties for the ones mentioning left and right)